posthog/hogql/bytecode.py

```diff
@@ -125,10 +184,91 @@ def visit_call(self, node: ast.Call):
             for arg in reversed(node.args):
                 args.extend(self.visit(arg))
             return [*args, Operation.OR, len(node.args)]
-        if node.name not in SUPPORTED_FUNCTIONS:
-            raise NotImplementedError(f"HogQL function `{node.name}` is not supported")
+        # TODO: do we want this check? gives better errors earlier
```

Comment:
removed this when we decided to remove `HOG_FUNCTIONS` (stl functions), but having this can give errors earlier while generating bytecode instead of while executing

Reply:
we should change the STL to be a dict like `{ "concat": concat_function }`, etc. Then we can check this easily.